sdk/keyvault/azure-security-keyvault-jca/src/main/java/com/azure/security/keyvault/jca/implementation/model/CertificateListResult.java

```diff
@@ -50,4 +55,43 @@ public void setNextLink(String nextLink) {
     }
 
     private String nextLink;
+
+    @Override
+    public JsonWriter toJson(JsonWriter jsonWriter) throws IOException {
+        jsonWriter.writeStartObject();
```

Comment:
Any reason `value` and `nextLink` aren't being serialized here?

Reply:
I copied this over from the Key Vault Certificates library where this type is read-only. I'll add those two here just in case, it's an implementation detail after all.